### ISSUE TITLE
Spawned Tails Randomly Decide Whether to Have a Bodypart sprite_datum or Not

### ISSUE THREAD

**Issue by wulfenbach-jpg** [tgstation/tgstation#76599]
## Reproduction:
(For the purposes of replication, this was done with 100 of each tail subtype in a local server. The local server was set to `LOWMEMORYMODE` on Runtime Station. Commit ID: 6ca75629abadbb45f1418e5d967156d455914a01)

1. Acquire any **subtype** of  `/obj/item/organ/external/tail`. More than 1 to replicate the issue.
a. This includes: `/obj/item/organ/external/tail/cat`, `/obj/item/organ/external/tail/monkey`, `/obj/item/organ/external/tail/lizard`, & `/obj/item/organ/external/tail/lizard/fake`.
b. This issue was replicated through the Game Panel verb. However, this may be replicated by acquiring these tails through **gibbing a simple mob**.
2. VV the tail on the ground.
3. Navigate to its `bodypart_overlay = /datum/bodypart_overlay/mutant/tail/cat` variable. This is usually the 37th in the list.
See the following:
![image](https://github.com/tgstation/tgstation/assets/53832839/c46f7c8b-886a-4988-bc9d-55203a5bbd06)
`/obj/item/organ/external/tail/cat`
![image](https://github.com/tgstation/tgstation/assets/53832839/6eb2d3d8-1794-43dd-87b6-624372a6c874)
`/obj/item/organ/external/tail/monkey`
![image](https://github.com/tgstation/tgstation/assets/53832839/c7aa2cee-4803-4f08-a486-c1b61f80096a)
`/obj/item/organ/external/tail/lizard`
![image](https://github.com/tgstation/tgstation/assets/53832839/d34ee9d2-f506-4565-aff9-388c6bda2753)
`/obj/item/organ/external/tail/lizard/fake`
4. Click on it.
5. Look at the `sprite_datum` variable. This is usually the 15th in the list. This appears to have a nonconstant value.

The following statistics were gathered by using SDQL_2 in the aforementioned local server.
### `/obj/item/organ/external/tail/cat`
![image](https://github.com/tgstation/tgstation/assets/53832839/34f78819-b7f5-4c0c-a396-19562b6dd5e4)
Out of 100 cat tails, 51 `sprite_datum`s have a proper overlay sprite.
![dreamseeker_06-01-17](https://github.com/tgstation/tgstation/assets/53832839/480ecc57-d278-4937-8584-ded8dadbda67)
The effects of a `None` value can be seen in the above figure. The bottom human has a `None` tail implanted.
I used the following SDQL command:
`SELECT /obj/item/organ/external/tail/cat IN (@[/area/station/hallway/primary/central MAP contents])[1] MAP bodypart_overlay MAP sprite_datum`
I then pasted the results of the list into Excel and graphed the resultant data.

### `/obj/item/organ/external/tail/monkey`
![image](https://github.com/tgstation/tgstation/assets/53832839/b9876125-cc9d-4e8e-9e04-e90714d3daa6)
![image](https://github.com/tgstation/tgstation/assets/53832839/7f7212f4-fc1b-448b-b348-5f432acfd8db)
Out of 100 monkey tails, 91 `sprite_datum`s have a proper overlay sprite. Only 9 display the `None` issue.
I used the following SDQL command:
`SELECT /obj/item/organ/external/tail/monkey  IN (@[/area/station/hallway/primary/central MAP contents])[1] MAP bodypart_overlay MAP sprite_datum`

### `/obj/item/organ/external/tail/lizard`
![image](https://github.com/tgstation/tgstation/assets/53832839/2cc6e329-3e73-4981-8ad9-b06b88294612)
![image](https://github.com/tgstation/tgstation/assets/53832839/1bbd3520-5b2c-43f5-a397-507115f7e10c)

Out of 100 lizard tails, 76 `sprite_datum`s have a proper overlay sprite. 24 display the `None` issue.
I used the following SDQL command:
`SELECT /obj/item/organ/external/tail/lizard  IN (@[/area/station/hallway/primary/central MAP contents])[1] MAP bodypart_overlay MAP sprite_datum`
To clarify, this was run after deleting all fake lizard tail subtypes from the testing area.

### `/obj/item/organ/external/tail/lizard/fake`
![image](https://github.com/tgstation/tgstation/assets/53832839/39c06be0-0bf6-40d5-909c-bfc3c060dfd0)
![image](https://github.com/tgstation/tgstation/assets/53832839/e8de3007-1162-432b-ba9e-be105846623a)
Out of 100 fake lizard tails, 76 `sprite_datum`s have a proper overlay sprite. 24 display the `None` issue.
I used the following SDQL command:
`SELECT /obj/item/organ/external/tail/lizard/fake  IN (@[/area/station/hallway/primary/central MAP contents])[1] MAP bodypart_overlay MAP sprite_datum`

### Regular Circumstances
For ~science~ the purposes of in-round reproduction, I spawned 100 `/mob/living/simple_animal/pet/cat`s and gibbed all of them. Results are outlined below.
![image](https://github.com/tgstation/tgstation/assets/53832839/22775be5-00f0-41f1-bcd7-418e52580bdb)
![image](https://github.com/tgstation/tgstation/assets/53832839/a214bdf5-caa6-4af3-997c-6e64b756ef00)

For good measure, I decided to spawn 100 felinids and gib them all as well.
![image](https://github.com/tgstation/tgstation/assets/53832839/3aaaaab9-214b-41f9-a57b-ab8d12b3fd1e)
![image](https://github.com/tgstation/tgstation/assets/53832839/aec150ce-b19e-4b8e-9511-ce8e5b22bb76)

It appears that tails found in `/mob/living/carbon/human` species appear to function properly. However, tails either admin spawned from game panel or acquired by gibbing a mob, likely any simple mob that has a subtype of tail in its `butcher_results`, causes an arcanely inconsistent lack of `sprite_datum`s.

**Comment by Sealed101** [comment]
related #76353

**Comment by ChungusGamer666** [comment]
This should be simple enough to fix...

**Comment by nikothedude** [comment]
holy shit i have never seen such a complete issue report this is beautiful completely completely beautiful

**Comment by Sealed101** [comment]
yeah the amount of effort here is spectacular i hope you're doing okay

**Comment by ChungusGamer666** [comment]
greatest bug report of all time
this is so excessively detailed i'm concerned for your mental health

**Comment by TheVekter** [comment]
This deserves fuckup of the year if just because it's probably the most effort I've ever seen someone put into an issue report for us.

**Comment by ChungusGamer666** [comment]
> This deserves fuckup of the year if just because it's probably the most effort I've ever seen someone put into an issue report for us.

time waste of the year 2023

**Comment by wulfenbach-jpg** [comment]
> > This deserves fuckup of the year if just because it's probably the most effort I've ever seen someone put into an issue report for us.
> 
> time waste of the year 2023

\>:(
i'll have you know I only spent two and a half hours on this !!!

**Comment by Kapu1178** [comment]
This is god's strongest issue reporter

**Comment by Sealed101** [comment]
history in the making
